Countly.m

```diff
@@ -287,7 +287,7 @@ - (CountlyConfig *) checkAndFixInternalLimitsConfig:(CountlyConfig *)config
     }
     
     if (config.sdkInternalLimits.getMaxValueSize == 0) {
-        [config.sdkInternalLimits setMaxKeyLength:kCountlyMaxValueSize];
```

Comment:
oh those were slipped from our eyes? or it was like that before

Reply:
Yes, from our eyes :)